docs/admin/deployment.rst

```diff
@@ -72,6 +72,18 @@ you create:
    Set to 'gitlab' if you want to use 'GitLab' (corresponding GITLAB_* settings must be set if required).
    Set to 'google' if you want to use 'Google' (corresponding GOOGLE_* settings must be set).
 
+``BADGES_PROMOTION_THRESHOLDS``
+   Optional. A comma-separated list of numeric thresholds for different levels of the 
```

Comment:
I'm not sure these explanations are clear without looking at the code.

For each of these options: 
* Do we support any number of levels? 
* Providing an example would also help, maybe with the default value.